### ISSUE TITLE
Unresolvable plocal/remote issue v2.2.0

### ISSUE THREAD

**Issue by anywhichway** [orientechnologies/orientdb#6198]
## Expected behavior and actual behavior

Expect: Access database through Studio and through custom browser app.
Actual:

```
2016-05-23 21:31:52:171 SEVER {db=Test} Internal server error:
com.orientechnologies.orient.core.exception.OFileLockedByAnotherProcessException: File 'C:\Users\Simon\OrientDB\orientdb-community-2.2.0\databases\Test\database.ocf' is locked by another process, maybe the database is in use by another process. Use the remote mode with a OrientDB server to allow multiple access to the same database [ONetworkProtocolHttpDb]
```

## Steps to reproduce the problem

Access database from Studio (success), attempt to use an app built with the browser orientdb.js library. Error in console.
Shutdown and restart database server
Access database from custom app (success), attempt to access database from Studio. Error in console.

Can't find any documentation that says how to either start the Studio in remote mode or use the JavaScript library in remote mode. Everything worked and still works fine in v1.1.7. Dug through the server.bat and xml config files to no avail.

Is this a bug in v2.2.0, an undocumented behavior change, or a bug in the documentation? Or am I just missing something?

### Runninng Mode
- [ x] Embedded, using PLOCAL access mode
- [ ] Embedded, using MEMORY access mode
- [ ] Remote (can't figure out how to get v2.2.0 to run in this mode for Studio)

### Misc
- [ ] I have a distributed setup with multiple servers. How many? 
- [ ] I'm using the Enterprise Edition

### OrientDB Version
- [ ] v2.0.x - Please specify last number: 
- [ ] v2.1.x - Please specify last number: 
- [0 ] v2.2.x - Please specify last number: 

### Operating System
- [ ] Linux
- [ ] MacOSX
- [x ] Windows
- [ ] Other Unix
- [ ] Other, name? 

### Java Version
- [ ] 6
- [ ] 7
- [x ] 8 64 bit u92

**Comment by anywhichway** [comment]
I did some more digging. This seems to be related to the version of Java installed, we went from 8 u42 x86 to 8 u94 x64. When we run OrientDB 2.1.17 on the newer version of Java it also fails with the same error, reverting to the older version of Java fixes the problem. Unfortunately 2.2.0 will not run on the older version of Java. I hope you guys can fix at your level though, can't find any JVM flags to address the issue.
